readme.txt

```diff
@@ -314,12 +315,12 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Dev: Revert work done in #4857 for automated shipping after OBW is completed #5971
 - Add: Welcome modal when coming from Calypso #6004
 - Enhancement: Add an a/b experiment for installing free business features #5786
-- Dev: Add `onChangeCallback` feature to the wc-admin <Form> component #5786 
```

Comment:
Can we revert all these newline changes?

Reply:
Done.

Phew... resolving newline conflicts from revert after your `merge main`, was the hardest part of this PR ;)